Source/Hurl.RulesetManager/MainWindow.xaml

```diff
@@ -0,0 +1,119 @@
+                x:Name="WarnText"
+                Height="0"
+                HorizontalAlignment="Center"
+                Foreground="Red"
+                Text="Atleast input Target Browser and Rules fields!"
+                Visibility="Hidden" />
+
+            <ui:Card Padding="10">
+                <Grid>
+                    <Grid.RowDefinitions>
+                        <RowDefinition Height="1*" />
+                        <RowDefinition Height="1*" />
+                        <RowDefinition Height="1*" />
+                    </Grid.RowDefinitions>
+                    <Grid.ColumnDefinitions>
+                        <ColumnDefinition Width="1*" />
+                        <ColumnDefinition Width="150" />
+                    </Grid.ColumnDefinitions>
+
+                    <!--<TextBlock VerticalAlignment="Center" Text="Rule Mode:" /> -->
+
+                    <TextBlock VerticalAlignment="Center" Text="Target Browser:" />
+                    <ComboBox
+                        x:Name="TargetBrowser"
+                        Grid.Column="1"
+                        Margin="0,0,0,10"
+                        ItemsSource="{Binding}" />
+                    <TextBlock
+                        Grid.Row="1"
+                        VerticalAlignment="Center"
+                        Text="Alternate Launch:" />
+                    <ComboBox
+                        x:Name="TargetBrowserx"
+                        Grid.Row="1"
+                        Grid.Column="1"
+                        ItemsSource="{Binding}" />
+                </Grid>
+            </ui:Card>
+
+            <ui:Card>
+                <StackPanel Orientation="Horizontal">
+                    <ComboBox
+                        x:Name="RuleModeInput"
+                        Margin="10"
+                        SelectedIndex="2">
+                        <ComboBoxItem>Regex</ComboBoxItem>
+                        <ComboBoxItem>Glob</ComboBoxItem>
+                        <ComboBoxItem>String</ComboBoxItem>
+                    </ComboBox>
+
+                    <ui:TextBox
+                        x:Name="RuleInputx"
+                        Margin="10"
+                        PlaceholderText="*.google.com" />
+
+                    <ui:Button>Add Rule</ui:Button>
+                </StackPanel>
+            </ui:Card>
+
+            <ui:Card Padding="10">
+                <StackPanel>
+                    <Grid Margin="0,0,0,10">
+                        <TextBlock VerticalAlignment="Center" Text="Type the Rules: (Seperate multiple rules with a pipe | and space )" />
+                        <!--<ui:Button HorizontalAlignment="Right">
+                        <ui:Button.Icon>
+                            <ui:SymbolIcon Symbol="Add24" />
+                        </ui:Button.Icon>
+                    </ui:Button>-->
+                    </Grid>
+                    <ui:TextBox
+                        x:Name="RuleInput"
+                        Width="460"
```

Comment:
I the mainwindow sizable? How does this react?

Reply:
Window should be resizable, for now for prototyping the functionality. I just took a random value in this case.